scripts/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+# Make sure submodule is registered and up-to-date.
+git submodule update --init
+
+# Apply the patch
+cd ckeditor-dev
+git checkout master
+git pull
+git checkout liferay
```

Comment:
Branch may not exist yet when person runs this after a fresh clone, so should probably check that first...

```sh
git branch -f liferay
git checkout liferay
```

Reply:
👍